migrations/0001_initial.py

```diff
@@ -306,7 +306,7 @@ class Migration(migrations.Migration):
                 ('description', models.TextField(max_length=1024, blank=True)),
                 ('starttime', models.DateTimeField(verbose_name=b'Start Time')),
                 ('endtime', models.DateTimeField(verbose_name=b'End Time')),
-                ('event', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, default=tracker.models.event.LatestEvent, to='tracker.Event')),
+                ('event', models.ForeignKey(on_delete=django.db.models.deletion.PROTECT, to='tracker.Event')),
```

Comment:
Is this an automatic change by the migration system, or just something you wanted to add in? I guess it won't really break anything, just curious.